guides/source/active_record_migrations.md

```diff
@@ -367,23 +367,24 @@ create_join_table :products, :categories
 which creates a `categories_products` table with two columns called
 `category_id` and `product_id`. These columns have the option `:null` set to
 `false` by default. This can be overridden by specifying the `:column_options`
-option.
+option:
 
 ```ruby
-create_join_table :products, :categories, column_options: {null: true}
+create_join_table :products, :categories, column_options: { null: true }
 ```
 
-will create the `product_id` and `category_id` with the `:null` option as
-`true`.
+which creates the columns `product_id` and `category_id` with the `:null` option
```

Comment:
Hello @r11runner – almost there. I think this additional paragraph is not required.
The sentence above with the example already makes it clear what happens when you specify `column_options`.
So you can just delete "which creates the columns `product_id` and `category_id` with the `:null` option as `true`."

That's all for my feedback. Please squash all your commits into one, and then your PR will be ready to get accepted.

Thanks!